src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1058,26 +1058,48 @@ def get_active_child_names(self):
         """Names of children that are currently active."""
         ret = []
         for child in self.child_names:
-            if getattr(self, child).is_active():
+            if getattr(self, child).is_active() and not self._is_deprecated(child):
```

Comment:
@prmukherj Deprecated items should already be inactive, right? So, it's `is_active` itself that would be updated.